db/seeds.rb

```diff
@@ -3,7 +3,7 @@
     photo: 'https://img.freepik.com/free-vector/mysterious-mafia-man-smoking-cigarette_52683-34828.jpg?w=1480&t=st=1674661161~exp=1674661761~hmac=e529680000ea966150e5ea3b38241a0d8c9582faf23dac8af61ce1785cf27838',
     bio: 'Fullstack Web developer from Uganda', posts_counter: 0)
     
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221368118-7a39757a-617c-47a3-b525-00a0bcdab326.png)
- [ ] It would be nice if you remove this section description shown in the image above to make your readme file descriptive and professional :blush: